packages/ui-kit-components/src/components/SplashScreen/splashScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import '@fontsource/merriweather/400.css';
```

Comment:
1) I don't think this import should be made, if the component is wrapped in a theme the font will have been imported by the theme - let me know if you have questions on this because it is a bit confusing

2) File name - practice is that React components all start with a capital letter so `SplashScreen` not `splashScreen`